.github/workflows/vercel-preview.yaml

```diff
@@ -0,0 +1,38 @@
+name: Deploy Vercel Preview
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+
+on:
+  pull_request: 
+    types: [ opened, reopened, edited, synchronize ]
+  workflow_dispatch:
```

Comment:
It looks like this only works for PRs against `gravitational/docs`. Is that the intention?

Since docs PRs already get a comment from the Vercel bot that looks like the one below, does this mean `gravitational/docs` PRs will get two comments with links to preview sites?

<img width="902" alt="image" src="https://github.com/gravitational/docs/assets/25250668/460606a1-49c9-4a8c-8c85-24b99065308d">